packages/ember-glimmer/tests/integration/helpers/unbound-test.js

```diff
@@ -0,0 +1,545 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { strip } from '../../utils/abstract-test-case';
+import { set } from 'ember-metal/property_set';
+import { get } from 'ember-metal/property_get';
+import setProperties from 'ember-metal/set_properties';
+import Component from 'ember-views/components/component';
+import EmberObject from 'ember-runtime/system/object';
+import { A as emberA } from 'ember-runtime/system/native_array';
+
+moduleFor('Helpers test: {{unbound}}', class extends RenderingTest {
+
+  ['@htmlbars it should not update the value if the parent view doesn\'t rerender']() {
+    this.render(`{{unbound foo}} {{unbound bar}}`, {
+      foo: 'BORK',
+      barBinding: 'foo'
+    });
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => set(this.context, 'foo', 'OOF'));
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => set(this.context, 'foo', 'BORK'));
+
+    this.assertText('BORK BORK');
+  }
+
+  ['@htmlbars should be able to output a property without binding']() {
+    this.render(`<div id="first">{{unbound content.anUnboundString}}</div>`, {
+      content: EmberObject.create({
+        anUnboundString: 'No spans here, son.'
+      })
```

Comment:
Nit: it doesn't seem like we will need an `EmberObject` here, as long as we go through `Ember.set` both would work the same